_posts/en/newsletters/2020-06-10-newsletter.md

```diff
@@ -0,0 +1,308 @@
+  q0="QUESTION"
+  a0="ANSWER"
+  a0link="/"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.20.0][] is the newest major release of Bitcoin's
+  oldest project.  This release's most notable user-visible improvements
+  include default bech32 addresses for RPC users (the GUI began
+  defaulting to bech32 in 0.19), the ability to configure RPC
+  permissions for different users and applications (see [Newsletter
+  #77][news77 rpcwhitelist]), some basic support for generating PSBTs in
+  the GUI ([#74][news74 psbtgui] & [#82][news82 psbtgui]), and a
+  `sortedmulti` addition to the [output script descriptor][topic
+  descriptors] language that can make it easier to watch multisig
+  addresses generated from lexicographical sorted keys, such as those
+  used with ColdCard's native multisig support ([#68][news68
+  sortedmulti]).
+
+    Many less visible improvements have also been made to the code in
+    order to eliminate bugs, enhance security, and prepare for future
+    changes.  One forward-looking change that's attracted some attention
+    is the addition of an `asmap` configuration setting that allows
+    using a separately-downloaded database to increase Bitcoin Core's
+    ability to avoid connecting to too many peers all running on a
+    network owned by the same group, increasing it's resistance to
+    [eclipse attacks][topic eclipse attacks] (see [Newsletter
+    #83][news83 asmap]).  However, one of the authors of the feature
+    [notes][wuille asmap], "it's highly experimental for now, and
+    unclear how to progress from that.  Gathering and compiling the
+    ASN [Autonomous Service Number] data is very nontrivial, and poses
+    trust questions."
+
+    For more information about all the changes as well as a list of over
+    100 people who contributed to this new version, see the project's
+    [release notes][core20 rn].
+
+- [LND 0.10.1-beta][] is a new minor release of this popular LN node
+  software.  The release notes announce no major new features but
+  do mention several bug fixes, a "new dry run migration mode to test
+  out migrations before they're applied permanently, and an enhancement
+  to the channel selection/restrictions in the router sub-server."
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Eclair #1440][] Accept multiple channels for some API FIXME:adamjonas
+
+- [Eclair #1141][] adds support for `option_static_remotekey` channels.
+  In the case of data loss on your end, this feature allows your channel
```

Comment:
Note: There are two places where the 3rd-person user is switched to 2nd-person "you" -- this paragraph, and the "Check hardware wallet compatibility" paragraph in Action Items.

Reply:
I'm changing this paragraph to third person, but the Action Items section is definitely meant to refer directly to the reader.